src/pages/my_page/user_profile/ResearchProfilePanel.tsx

```diff
@@ -15,21 +12,15 @@ import { PageSpinner } from '../../../components/PageSpinner';
 import { BackgroundDiv } from '../../../components/styled/Wrappers';
 import { Helmet } from 'react-helmet-async';
 
-export const ResearchProfilePanel = () => {
+interface ResearchProfilePanelProps {
+  person: CristinPerson | undefined;
```

Comment:
Sparer noen tastetrykk :D
```suggestion
  person?: CristinPerson;
```